lib/models/addon-discovery.js

```diff
@@ -146,11 +146,7 @@ class AddonDiscovery {
       }
 
     } catch (e) {
-      let acceptableError = `Cannot find module '${packageName}' from '${root}'`;
-      // pending: https://github.com/substack/node-resolve/pull/80
-      let workAroundError = 'Cannot read property \'isFile\' of undefined';
-
-      if (e.message === workAroundError || e.message === acceptableError) {
+      if (e.code === 'MODULE_NOT_FOUND') {
```

Comment:
If you wanted to keep the old check too, then you'd remain compatible with < v1.3 of resolve